### PR TITLE
[PLAT-71] Pass volume variable to tf_ecs_task_definition

### PR DESCRIPTION
**Pull request by keymon**
As per https://github.com/mergermarket/tf_ecs_task_definition/pull/3, pass the volume variable to the underlying repository.

We also add the missing output arn.

Dependencies
------------

**IMPORTANT** merge https://github.com/mergermarket/tf_ecs_task_definition/pull/3 then remove the temporary commit pointing to that branch in this repo before merge.

### CODE REVIEW

File: test/infra/main.tf [hunk 1]
```diff
@@ -10,6 +10,12 @@ provider "aws" {
   region                      = "eu-west-1"
 }
 
+variable "task_volume_param" {
+    description = "Allow the test to pass this in"
```
Comment (bart613) [inline test/infra/main.tf]: Would it be useful to elaborate on this description a bit more?

File: test/test_taskdef_with_role.py [hunk 2]
```diff
@@ -1,4 +1,7 @@
 import unittest
```
Comment (bart613) [inline test/test_taskdef_with_role.py]: This bit could do with a bit of [isorting](https://github.com/timothycrosley/isort)

File: test/test_taskdef_with_role.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+import os
```
Comment (bart613) [inline test/test_taskdef_with_role.py]: isort

File: variables.tf [hunk 4]
```diff
@@ -12,3 +12,9 @@ variable "policy" {
     description = "A valid IAM policy for the task"
     type        = "string"
 }
+
+variable "volume" {
+    description = "Volume block map with 'name' and 'host_path'. 'name': The name of the volume as is referenced in the sourceVolume. 'host_path' The path on the host container instance that is presented to the container."
```
Comment (bart613) [inline variables.tf]: I'd probably shorten the description here a bit and refer to README for full guide on how to use this; just something to consider

**Review by bart613 — CHANGES_REQUESTED** [review]
Solid stuff - few little tweaks suggestions for consideration but looks good 👍

**Comment by keymon** [comment]
@bart613 I applied the changes as suggested :+1: :)

**Review by bart613 — APPROVED** [review]

**Comment by bart613** [comment]
Thanks - ready to go :shipit:

**Comment by keymon** [comment]
Cool, @bart613. If you approve https://github.com/mergermarket/tf_ecs_task_definition/pull/3 I would merge that one and remove the temporary branch reference here.

**Pull request merged**